LocalRunner.js

```diff
@@ -20,12 +21,15 @@ class LocalRunner {
     const dir = path.join(this.rootDir, instance.name);
     const dataDir = path.join(dir, 'data');
     const appsDir = path.join(dir, 'apps');
+    const logFile = path.join(dir, 'arangod.log');
+    instance.logFile = logFile;
 
     instance.args.unshift('--configuration=none');
     instance.args.push(...[
       `--javascript.startup-directory=${path.join(this.basePath, 'js')}`,
       `--javascript.app-path=${appsDir}`,
       `--server.endpoint=${instance.endpoint}`,
+      `--log.file=${logFile}`,
```

Comment:
Do these require any changes to the docker runner (if that one isn't completely broken anyway right now)?

Reply:
@pluma It seems to me the docker runner is broken since b11c06acfcba175f9be751214404cd75b3f1bb1f. I can run the tests with 7b1a6dac7bb17099fb484a66aaf054fbab675481 (the commit right before), but with none after that, so it may be broken for a year.

If someone can confirm that the docker tests in master are broken I'd vote to ignore this here and fix the docker tests in another PR.